style.css

```diff
@@ -507,3 +518,157 @@ section {
   background-color: #fff;
   border-radius: 8px;
 }
+
+@media only screen and (min-width: 768px) {
+  body {
+    overflow-x: hidden;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    visibility: visible;
+    gap: 32px;
+    position: relative;
+    bottom: 15px;
+    right: 10.76%;
+  }
+
+  .nav-links a {
+    width: 46px;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .navigation,
+  .bio-section,
+  .about-section,
+  .contact-me {
+    width: 100vw;
+    overflow-x: hidden;
+  }
+
+  .bio-section {
+    height: 100vh;
+    position: relative;
+    top: -80px;
+    background-image: url(img/desk-bg.png);
+    background-size: 100% auto;
+  }
+
+  .bio-section .content {
+    padding: 10%;
+    width: 80vw;
+    position: relative;
+    top: 4%;
+  }
+
+  .bio-header {
+    width: 44vw;
+  }
+
+  .bio-text {
+    font-size: 1rem;
+  }
+
+  .cards {
+    display: flex;
+    flex-direction: column;
+    gap: 88px;
+  }
+
+  .card {
+    width: 80.2vw;
+    height: 70%;
+    position: relative;
+    left: 9.86%;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .card-img img {
+    height: 172%;
+    width: 102%;
+    position: relative;
+    top: -39%;
+    left: -5%;
+  }
+
+  .card-items {
+    width: 99%;
+    height: 81%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 12%;
+    position: relative;
+    left: 4%;
+    top: -16%;
+  }
+
+  .card-props {
+    margin-right: 7%;
+  }
+
+  .oppo {
+    left: 88%;
+  }
+
+  #oppo,
+  #oppo2 {
+    position: relative;
+    left: 1%;
+    top: -15%;
+  }
+
+  .about-section .content {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    margin-top: 21%;
+  }
+
+  .about-2 {
+    position: relative;
+    top: 10%;
+    gap: 21%;
+  }
+
+  .lang {
+    flex-direction: row;
+    gap: 26%;
+  }
+
+  .lan {
+    width: 12%;
+  }
+
+  .lan img {
+    height: 71%;
+  }
+
+  .lan p,
+  .lefter-1,
+  .lefter-2 {
+    position: relative;
+    top: 57%;
+    right: 119%;
+    margin: 10%;
+  }
+
+  .contact-me {
+    background-image: url(img/desk-bg.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: inherit;
+    background-position: initial;
+  }
+
+  .contact-form button {
+    margin-left: 29%;
+  }
+}
```

Comment:
please I'd suggest you implement these changes for your contact-me section for your project  to have a better user experience:

- [ ] kindly add the background images to be as it is in the figma template
- [ ] please use the right margins and alignment for your `contact- me` text and input boxes as well as their width
- [ ] Also use the right button styling for your buttons

Your design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/e88388ff-6d4d-4ac4-a32d-4ec9204a8c31)

Figma design :
![image](https://github.com/josman227/Mobile-Portfolio/assets/20537712/f3818b9e-6ea5-4dd2-b41b-f55cc9393c0d)